cl/recap/tests.py

```diff
@@ -7,11 +7,13 @@
 
 from django.conf import settings
 from django.contrib.auth.models import User
+from django.core import mail
```

Comment:
In this file, can you verify that we need all the new json fixtures? I'm surprised we don't have enough already, but I haven't paid close attention. 

Second, can you check that your mocks are all necessary? Perhaps some can be replaced with lambdas?